### PR TITLE
Source Google Ads: update input field tooltips

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
Made some small updates to the tooltip descriptions for Google Ads

## How
- Removed comment in `customer_id` field about not being able to request metrics streams for Manager accounts as it doesn't seem pertinent to explaining the field's value
- Added a pattern descriptor for `login_customer_id` field
- Noted that `conversion_window` uses days as the time unit
- Minor edits and typo fixes

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`2a2daacea0`](https://github.com/airbytehq/airbyte/pull/28832/commits/2a2daacea011b1e79f1ff815c58811cbf68a0899)) - ✅

⏲️  Total pipeline duration: 27mn44s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-ads-spec-update/1690568100/2a2daacea011b1e79f1ff815c58811cbf68a0899/source-google-ads/0.7.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json [hunk 1]
```diff
@@ -97,31 +97,32 @@
               "type": "string",
               "multiline": true,
               "title": "Custom Query",
-              "description": "A custom defined GAQL query for building the report. Should not contain segments.date expression because it is used by incremental streams. See Google's <a href=\"https://developers.google.com/google-ads/api/fields/v11/overview_query_builder\">query builder</a> for more information.",
+              "description": "A custom defined GAQL query for building the report. Avoid including the segments.date expression as it is used by incremental streams. For more information, refer to <a href=\"https://developers.google.com/google-ads/api/fields/v11/overview_query_builder\">Google's documentation</a>.",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json:100]:
```suggestion
              "description": "A custom defined GAQL query for building the report. Avoid including the segments.date field; wherever possible, Airbyte will automatically include it for incremental syncs. For more information, refer to <a href=\"https://developers.google.com/google-ads/api/fields/v11/overview_query_builder\">Google's documentation</a>.",
```

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json [hunk 2]
```diff
@@ -97,31 +97,32 @@
...
             }
           }
         }
       },
       "login_customer_id": {
         "type": "string",
         "title": "Login Customer ID for Managed Accounts",
-        "description": "If your access to the customer account is through a manager account, this field is required and must be set to the customer ID of the manager account (10-digit number without dashes). More information about this field you can see <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">here</a>",
+        "description": "If your access to the customer account is through a manager account, this field is required, and must be set to the 10-digit customer ID of the manager account. For more information about this field, refer to <a href=\"https://developers.google.com/google-ads/api/docs/concepts/call-structure#cid\">Google's documentation</a>.",
+        "pattern_descriptor": "The customer ID must be 10 digits, with no dashes.",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json:117]: the UI already adds a prefix to pattern_descriptors https://docs.airbyte.com/connector-development/connector-specification-reference/#pattern-descriptors so this should just describe the pattern
```suggestion
        "pattern_descriptor": ": 10 digits, with no dashes.",
```

**Review by sherifnada — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`4d394e599e`](https://github.com/airbytehq/airbyte/pull/28832/commits/4d394e599ec02783b4beffba3f50100c72f3a0f2)) - ✅

⏲️  Total pipeline duration: 18mn49s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-ads-spec-update/1690823083/4d394e599ec02783b4beffba3f50100c72f3a0f2/source-google-ads/0.7.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`3fdf2a7be0`](https://github.com/airbytehq/airbyte/pull/28832/commits/3fdf2a7be028db31128ecedfb3d96c93250e4021)) - ❌

⏲️  Total pipeline duration: 25mn06s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-ads-spec-update/1690824364/3fdf2a7be028db31128ecedfb3d96c93250e4021/source-google-ads/0.7.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-ads test report (commit [`dbc169dfa2`](https://github.com/airbytehq/airbyte/pull/28832/commits/dbc169dfa23af6d2df9c699861991c1abef569b1)) - ✅

⏲️  Total pipeline duration: 22mn52s 

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-ads/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-google-ads docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Acceptance tests                                                         | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/google-ads-spec-update/1690837431/dbc169dfa23af6d2df9c699861991c1abef569b1/source-google-ads/0.7.4/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connector_ops/connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-ads test
```

**Pull request merged**